src/cmd/deploy.go

```diff
@@ -50,69 +50,80 @@ type DeployEvent struct {
+	Long:  "Create deployment events (report a deployment to OpsLevel using an integration url)",
 	Run: func(cmd *cobra.Command, args []string) {
 		if integrationUrl == "" {
 			log.Error().Msg("Please provide '--integration-url' to send the deployment information to")
 			os.Exit(1)
 		}
 
-		var err error
 		evt, err := readCreateConfigAsDeployEvent()
 		cobra.CheckErr(err)
 		if dryrun := viper.GetBool("dry-run"); dryrun {
 			b, _ := json.Marshal(evt)
 			log.Info().Msgf("%s", string(b))
-		} else {
-			body, err := json.Marshal(evt)
-			cobra.CheckErr(err)
-			response := &opslevel.RestResponse{}
-			resp, err := getClientRest().R().
-				SetHeader("Content-Type", "application/json").
-				SetBody(body).
-				SetResult(response).
-				Post(integrationUrl)
-			cobra.CheckErr(err)
-			if resp.IsSuccess() {
-				log.Info().Msgf("Successfully registered deploy event for '%s'", evt.Service)
-			} else {
-				log.Error().Msg(resp.String())
-				os.Exit(1)
-			}
+			return
 		}
+		body, err := json.Marshal(evt)
+		cobra.CheckErr(err)
+		response := &opslevel.RestResponse{}
+		resp, err := getClientRest().R().
+			SetHeader("Content-Type", "application/json").
+			SetBody(body).
+			SetResult(response).
+			Post(integrationUrl)
+		cobra.CheckErr(err)
+		if !resp.IsSuccess() {
+			log.Error().Msg(resp.String())
+			os.Exit(1)
+		}
+		log.Info().Msgf("Successfully registered deploy event for '%s'", evt.Service)
 	},
 }
 
 func init() {
 	createCmd.AddCommand(deployCreateCmd)
 
-	deployCreateCmd.Flags().StringVarP(&integrationUrl, "integration-url", "i", "", "OpsLevel integration url (OPSLEVEL_INTEGRATION_URL)")
 	deployCreateCmd.Flags().Bool("dry-run", false, "if true data will be logged and not sent to the integration-url (OPSLEVEL_DRY_RUN)")
-	deployCreateCmd.Flags().String("git-path", "./", "relative path to grab the git commit info from (if git repo is found overrides all commit details)")
-
-	deployCreateCmd.Flags().StringP("service", "s", "", "service alias for the event (OPSLEVEL_SERVICE)")
-	deployCreateCmd.Flags().StringP("description", "d", "", "description of the event (OPSLEVEL_DESCRIPTION)")
-	deployCreateCmd.Flags().StringP("environment", "", "", "environment name of the event (OPSLEVEL_ENVIRONMENT)")
-	deployCreateCmd.Flags().StringP("deploy-number", "", "", "deploy number of the event (OPSLEVEL_DEPLOY_NUMBER)")
-	deployCreateCmd.Flags().String("deploy-url", "", "url the event will link back to (OPSLEVEL_DEPLOY_URL)")
-	deployCreateCmd.Flags().String("dedup-id", "", "dedup id of the event (OPSLEVEL_DEDUP_ID)")
+	viper.BindEnv("dry-run", "OPSLEVEL_DRY_RUN", "OL_DRY_RUN")
 
-	deployCreateCmd.Flags().String("deployer-name", "", "deployer name who created the event (OPSLEVEL_DEPLOYER_NAME)")
-	deployCreateCmd.Flags().String("deployer-email", "", "deployer email who created the event (OPSLEVEL_DEPLOYER_EMAIL)")
+	deployCreateCmd.Flags().String("commit-message", "", "git commit message associated with the event (OPSLEVEL_DEPLOYER_EMAIL)")
```

Comment:
Organized `init()` so that each CLI flag is right beside the corresponding env var binding.